.github/workflows/release.yml

```diff
@@ -8,199 +8,16 @@ on:
 env:
   CI: true
 jobs:
-  prepare-deployment:
-    runs-on: ubuntu-20.04
-    outputs:
-      deployment-id: ${{ fromJson(steps.create-deployment.outputs.data).id }}
-    steps:
-      - name: Create GitHub Deployment
-        id: create-deployment
-        uses: octokit/request-action@v2.x
-        with:
-          route: POST /repos/:repository/deployments
-          repository: ${{ github.repository }}
-          ref: ${{ github.sha }}
-          environment: review
-          transient_environment: true
-          auto_merge: false
-          mediaType: '{"previews": ["flash", "ant-man"]}'
-          # The deployment runs in parallel with CI, so status checks will never have succeeded yet:
-          required_contexts: "[]"
-        env:
-          GITHUB_TOKEN: "${{ secrets.GH_TOKEN }}"
-
   publish-npm:
-    runs-on: ubuntu-20.04
-    needs: [prepare-deployment]
+    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: Mark GitHub Deployment as in progress
-        id: start-deployment
-        uses: octokit/request-action@v2.x
-        with:
-          route: POST /repos/:repository/deployments/:deployment/statuses
-          repository: ${{ github.repository }}
-          deployment: ${{ needs.prepare-deployment.outputs.deployment-id }}
-          environment: review
-          description: "Publishing to npm tag [${GITHUB_REF#refs/tags/v}]…"
-          log_url: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
-          state: in_progress
-          mediaType: '{"previews": ["flash", "ant-man"]}'
-        env:
-          GITHUB_TOKEN: "${{ secrets.GH_TOKEN }}"
       - name: Prepare for publication to npm
         uses: actions/setup-node@v3
         with:
-          node-version: "16.x"
+          node-version-file: ".nvmrc"
           registry-url: "https://registry.npmjs.org"
       - run: npm ci
-      - name: Publish to npm
-        run: |
-          npm publish
-          echo "Package published. To install, run:"
-          echo ""
-          echo "    npm install @inrupt/solid-client-vc"
+      - run: npm publish --access public
```

Comment:
Is it necessary to specify the `--access public` flag when the package already has been published as public at least once?

Reply:
Probably not, but easier to keep it